systems/game-servers/modules/tgs/EventScripts/tg/PreCompile.sh

```diff
@@ -74,8 +74,11 @@ cp target/i686-unknown-linux-gnu/release/libauxcpu_byondapi.so "$1/libauxcpu_byo
 cd "$work_directory"
 echo "auxcpu: deployment finish"
 
+# Get unzip
+apt-get install -y unzip
```

Comment:
you'll need this bc bun install requires it

Reply:
No apt in nix, this needs to be installed via the configs.

Reply:
there's also packages for tgs in that same file, specifically node, [L124](https://github.com/tgstation-operations/infrastructure/blob/main/systems/game-servers/modules/tgs/default.nix#L10). this seems more precise?

i'm still confused if this line is needed for it to run on tgstation ci

Reply:
That's the right spot. The one oranges linked isn't

Reply:
i'm going to remove the apt get line. i just wonder if this means it won't work on tgstation CI.